### PR TITLE
Armis activities

### PR DESCRIPTION
**Pull request by jayeshprajapaticrest**
Required items, please complete
   
   Change(s):
   - Added artifacts for Armis Activities Data connector

   Reason for Change(s):
   - New implementation

   Version Updated:
   - Initial Version

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes

-----------------------------------------------------------------------------------------------------------

### CODE REVIEW

**Comment by jayeshprajapaticrest** [comment]
@v-spadarthi 
Can you please let us know about the latest updates on this PR merge?

File: DataConnectors/ArmisActivities/ArmisActivitySentinelConnector/__init__.py [hunk 1]
```diff
@@ -0,0 +1,451 @@
...
+    )
+    logging.info(
+        "Armis Activity Connector: Python timer trigger function ran at %s",
+        utc_timestamp,
+    )
+
+    armis_obj = ArmisActivity()
+    try:
+        armis_obj.check_data_exists_or_not_activity()
+    except ArmisDataNotFoundException:
```
[the remark this answers is not in the document]
Comment (jayeshprajapaticrest) [reply DataConnectors/ArmisActivities/ArmisActivitySentinelConnector/__init__.py:440]: If no activities data found from Armis then exception is raised from the child method then we need actual function app trigger result as success in the FunctionApp -> Monitor

**Comment by jayeshprajapaticrest** [comment]
@v-spadarthi 
Can you please let us know about the latest updates on this PR merge?

**Comment by jayeshprajapaticrest** [comment]
@v-spadarthi 
Can you please let us know about the updates on this PR merge?

**Comment by jayeshprajapaticrest** [comment]
@v-mchatla 
I have added sample data for ArmisActivities parser at below location.
https://github.com/jayeshprajapaticrest/Azure-Sentinel/blob/ArmisActivities/Sample%20Data/ArmisActivities.csv

**Comment by NikTripathi** [comment]
Thanks @jayeshprajapaticrest .
@v-mchatla Please have a look.

**Comment by v-mchatla** [comment]
Hi @jayeshprajapaticrest
Thanks for sharing the data. I have verified the Parser and Data connector; changes look good to me.
Please fix the below things.

1. To maintain consistency, we are following this [Folder Structure](https://github.com/Azure/Azure-Sentinel/tree/master/Solutions/Box). Can you also follow the same folder structure for your content.
2. I could see there were few Azure Sentinel keywords in your data connector file, can you please update those with Microsoft Sentinel. As part of our new naming convention, we are using Microsoft Sentinel instead of Azure Sentinel.
3. Please follow respective folder and naming convention in readme and all other files wherever required
4. Please add metadata in the data connector file.
Let me know if you need any help over here.
Thanks

**Comment by v-mchatla** [comment]
> > Hi @jayeshprajapaticrest Thanks for sharing the data. I have verified the Parser and Data connector; changes look good to me. Please fix the below things.
> > 
> > 1. To maintain consistency, we are following this [Folder Structure](https://github.com/Azure/Azure-Sentinel/tree/master/Solutions/Box). Can you also follow the same folder structure for your content.
> > 2. I could see there were few Azure Sentinel keywords in your data connector file, can you please update those with Microsoft Sentinel. As part of our new naming convention, we are using Microsoft Sentinel instead of Azure Sentinel.
> > 3. Please follow respective folder and naming convention in readme and all other files wherever required
> > 4. Please add metadata in the data connector file.
> >    Let me know if you need any help over here.
> >    Thanks
> 
> @v-mchatla I have done changes of keywords. Also, since ArmisActivities, ArmisAlerts and ArmisDevice data connectors are of same organization, we have created a solution and added these connectors as part of that solution. We have raised a new [PR](https://github.com/Azure/Azure-Sentinel/pull/6151) for certification with all the three connectors and its parser with sample data.

Hi @jayeshprajapaticrest,
Thanks for accommodating the changes requested in the new PR. As you have already added all the changes in the PR's https://github.com/Azure/Azure-Sentinel/pull/5782, https://github.com/Azure/Azure-Sentinel/pull/5784, https://github.com/Azure/Azure-Sentinel/pull/5785 into new PR https://github.com/Azure/Azure-Sentinel/pull/6151. Can we close the existing PRs and track the status under single PR https://github.com/Azure/Azure-Sentinel/pull/6151.
Thanks

**Comment by jayeshprajapaticrest** [comment]
> Hi @jayeshprajapaticrest, Thanks for accommodating the changes requested in the new PR. As you have already added all the changes in the PR's #5782, #5784, #5785 into new PR #6151. Can we close the existing PRs and track the status under single PR #6151. Thanks

@v-mchatla 
Sure we close #5782 PR and Please update us on the new [PR](https://github.com/Azure/Azure-Sentinel/pull/6151) as we need this integration live as soon as possible. Thanks

**Comment by v-spadarthi** [comment]
@jayeshprajapaticrest : Can you please close this PR because you already raised another [PR̆](https://github.com/Azure/Azure-Sentinel/pull/6151) for the same

**Comment by jayeshprajapaticrest** [comment]
> @jayeshprajapaticrest : Can you please close this PR because you already raised another [PR̆](https://github.com/Azure/Azure-Sentinel/pull/6151) for the same

@v-spadarthi 
Sure I am going to close this, but please proceed as fast as you can with this [PR](https://github.com/Azure/Azure-Sentinel/pull/6151) as its been around 2 Month for this current PR for merging and we raised new [PR](https://github.com/Azure/Azure-Sentinel/pull/5782) just to maintain folder structure.

**Pull request closed without merging**